modules/meeting/app/components/meeting_agenda_items/new_button_component.html.erb

```diff
@@ -23,7 +23,7 @@
       )
       unless @meeting_section
         component.with_item(
-          label: "Section",
+          label: t("activerecord.models.meeting_section", count: 1),
```

Comment:
Why is the `count: 1` added here? The translation key you added does not have the `one`/`other` subkeys, so the count does not do anything here. 

```suggestion
          label: t("activerecord.models.meeting_section"),
```